docs/topics/db/fixtures.txt

```diff
@@ -126,11 +126,31 @@ You could also write a decorator to encapsulate this logic::
 Just be aware that this logic will disable the signals whenever fixtures are
 deserialized, not just during ``loaddata``.
 
-Note that the order in which fixture files are processed is undefined. However,
-all fixture data is installed as a single transaction, so data in
-one fixture can reference data in another fixture. If the database backend
-supports row-level constraints, these constraints will be checked at the
-end of the transaction.
+Fixtures can be loaded in a specified order. For example:
+
+.. code-block:: shell
+
+    manage.py loaddata foo bar baz
+
+or (in a test case)
+
+.. code-block:: shell
+
+    Fixtures = ['foo', 'bar', 'baz']
```

Comment:
If I were writing this, I might use more real-sounding names, like `people`, `places`, `events` - I find that helps me imagine an example better.

Reply:
I agree with this. Following the existing examples, we could do: `mydata`, `otherdata`, `thirdset` or similar.

Reply:
I also think that this example should include the class definition, otherwise is unclear where the line would show up in a test case.

Reply:
To summarize above comments, I suggest something like this as the code example:

.. code-block:: python

  class AnimalTestCase(TestCase):
      fixtures = ["mammals", "birds", "insects"]